.github/workflows/pypi-cd.yml

```diff
@@ -187,13 +186,13 @@ jobs:
               with:
                   working-directory: ./python
                   target: ${{ matrix.build.TARGET }}
-                  args: --release --strip --out wheels -i  ${{ github.event_name != 'pull_request' && 'python3.8 python3.9 python3.10 python3.11 python3.12' || 'python3.10' }}
+                  args: --release --strip --out wheels -i  ${{ github.event_name != 'pull_request' && 'python3.9 python3.10 python3.11 python3.12' || 'python3.10' }}
 
             - name: Upload Python wheels
               if: github.event_name != 'pull_request'
-              uses: actions/upload-artifact@v3
+              uses: actions/upload-artifact@v4
```

Comment:
`v3` is outdated. new version requires unique name, so I updated download step too